docs/valkyrie/odin/deployment/aws/server.md

```diff
@@ -0,0 +1,22 @@
+The Odin server is deployed in the AWS ec2 with a public endpoint. 
+
+## Prerequisites
+- Caddy installed and running on your system
+- Golang installed
```

Comment:
we can switch installation of binary to nix based since it gets all the dependencies also